ui/src/hooks/useAssets.ts

```diff
@@ -82,11 +84,11 @@ export default ({
       label: t("Z To A"),
     },
     {
-      value: `${AssetOrderBy.Size}_${OrderDirection.Desc}`,
+      value: `${AssetOrderBy.Size}_${OrderDirection.Asc}`,
```

Comment:
I believe current main is displaying correctly, so I think `value` is correct. Any reason you have switched this?